src/cpl/nuopc/lnd_comp_nuopc.F90

```diff
@@ -1038,53 +1041,54 @@ subroutine ModelSetRunClock(gcomp, rc)
        call ESMF_LogWrite(subname//'setting alarms for ' // trim(name), ESMF_LOGMSG_INFO)
 
        !----------------
-       ! Restart alarm
```

Comment:
@jedwards4b here the order of setting the stop alarm and then restart alarm was switched. As far as I could see, there isn't a strict need to do this. I figured a preferred order might be stop first and then restart, maybe to be consistent elsewhere.

But, I wanted to make sure I wasn't missing anything. So is this a preferential change or one that's absolutely needed? Thanks in advance.

Reply:
Actually there is a requirement to do this.  When you request to write the restart at the end of the run, you need to know when the end of run is so that you can set the restart alarm, by initializing the stop alarm first I have the information I need to set the restart alarm.

Reply:
Excellent, thanks for the explanation that helps.

I'll add a comment about this then. And make sure the same is done in LILAC.

Reply:
Moving this as a thing to do in https://github.com/ESCOMP/CTSM/pull/2912